File: Assignments/Assignment_6/program3.c
// Write a program which accept two numbers and check whether numbers are equal or not.

#include<stdio.h>
#include<stdbool.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : ChkEqual
//  Description :   It is used to display number is equal or not
//  Input :         Integer,Integer
//  Output :        boolean
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

int ChkEqual(int iNo1,int iNo2)
{
    if(iNo1 == iNo2)
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;
    bool bRet = false;

    printf("Please enter two numbers\n");
    scanf("%d %d",&iValue1, &iValue2);

    bRet = ChkEqual(iValue1,iValue2);

    if(bRet == true)
    {
        printf("Equal");
    }
    else
    {
        printf("Not Equal");
    }

    return 0;
}

// TIme Complexity : O(1)

File: Assignments/Assignment_17/program3.java
// Write a program to check whether a number is a number is palindrome or not.

class Logic
{
    void findMax(int a, int b)
    {

        if(a > b)
        {
        System.out.print("a is Maximum");
        }
        else
        {
            System.out.print("b is Maximum");
        }
    }
}

class program3
{
    public static void main(String args[])
    {
        Logic obj = new Logic();
        obj.findMax(20, 15);
    }

}